CefSharp.WinForms.Example/BrowserTabUserControl.cs

```diff
@@ -434,5 +434,42 @@ private void FindCloseButtonClick(object sender, EventArgs e)
         {
             ToggleBottomToolStrip();
         }
+
+        public void ShowDevToolsDocked()
+        {
+            if (browserSplitContainer.Panel2Collapsed)
+            {
+                browserSplitContainer.Panel2Collapsed = false;
+
+                if (devToolsPanel == null || devToolsPanel.IsDisposed)
+                {
+                    devToolsPanel = new Panel()
+                    {
+                        Dock = DockStyle.Fill
+                    };
+                    browserSplitContainer.Panel2.Controls.Add(devToolsPanel);
```

Comment:
Running this multiple times are we leaving `Disposed` panels in the `Controls` collection?

Reply:
Good point! This is highly likely every disposed `Panel` will exist the collection.
Will check this out!